compiler/il/ILOpCodeProperties.hpp

```diff
@@ -3474,11 +3474,11 @@
    /* .opcode               = */ TR::acmpeq,
    /* .name                 = */ "acmpeq",
    /* .properties1          = */ ILProp1::BooleanCompare,
-   /* .properties2          = */ ILProp2::ValueNumberShare | ILProp2::SupportedForPRE,
+   /* .properties2          = */ ILProp2::ValueNumberShare | ILProp2::UnsignedCompare | ILProp2::SupportedForPRE,
    /* .properties3          = */ ILProp3::CompareTrueIfEqual,
    /* .properties4          = */ 0,
    /* .dataType             = */ TR::Int32,
-   /* .typeProperties       = */ ILTypeProp::Size_4 | ILTypeProp::Integer,
+   /* .typeProperties       = */ ILTypeProp::Size_4 | ILTypeProp::Integer | ILTypeProp::Unsigned,
```

Comment:
Right, I did not think about that. Removing that should not affect the functionality, I'll do a build and remove the typeProperty if it passes.